stable/contour/templates/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: extensions/v1beta1
```

Comment:
Please use the `apps/v1` API instead. the extensions one for Deployment will soon not be accepted by the Kubernetes API as it is long past the deprecation window.

Reply:
changed to apps/v1